bin/build-content-json.js

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
```

Comment:
This script is kind of a temporary hack: `frontend/src/app/store.js` imports `../../build/api/{experiments,news_updates}.json`. That causes chicken-and-egg problem, because those JSON assets need to be built before everything else can be bundled.

Running this script first takes care of that. Once HTML generation moves into webpack, we should be able to import YAML directly in store.js and no longer need this.